src/include/daos_task.h

```diff
@@ -191,6 +192,16 @@ struct daos_obj_register_class_t {
 	struct daos_oclass_attr	*cattr;
 };
 
+/** pool management pool list args */
+typedef struct {
+	/** Process set name of the DAOS servers managing the pool */
+	const char            *grp;
```

Comment:
Agreed that it should either be "the system" or "the pools"... Was a copy/paste from other struct defs. I've changed it to "the system". With regard to the "process set" terminology, it's consistent with the other doc strings. I'm inclined to have it match the others, and if someone wants to mass-update everything, this will be included in that effort.